SenderEmail/pom.xml

```diff
@@ -57,6 +57,11 @@
 			<artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
 			<version>2.5.0</version>
 		</dependency>
+		<dependency>
+			<groupId>io.github.cdimascio</groupId>
```

Comment:
add command before and after dependency why i use this  
== START SECURE DATA =====
<dependency>
    <groupId>io.github.cdimascio</groupId>
    <artifactId>dotenv-java</artifactId>
    <version>2.2.0</version>
</dependency>
== END SECURE DATA ===========